### PR TITLE
Jacob Arellano

### PR DESCRIPTION
**Pull request by jakejakearell**

### CODE REVIEW

File: lib/school.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+class School
+  attr_reader :start_time, :hours_in_school_day, :student_names
+  def initialize(start_time, hours_in_school_day)
+    @start_time = start_time
+    @hours_in_school_day = hours_in_school_day
+    @student_names = []
+
```
Comment (robertguturing) [inline lib/school.rb:7]: Nitpick, extraline here

File: lib/school.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    @student_names = []
+    
+  end
+
+  def add_student_name(name)
+    @student_names << name
+  end
+
+  def end_time
+    int_start_time = @start_time.to_i
```
Comment (robertguturing) [inline lib/school.rb:15]: Nitpick: You shouldn't abbreviate words as it can confuse other developers that have to read your code.  Take the time to spell out `initial` here for developer empathy ❤️

File: lib/school.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    endtime = "#{(int_start_time+hours_in_school_day)}:00"
+  end
+
+  def is_full_time?
```
Comment (robertguturing) [inline lib/school.rb:19]: This is correct, but you can probably ditch the `if else` statement and instead have

`@hours_in_school_day > 4`

File: lib/school.rb [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    end
+    names
+
+  end
+
+  def convert_end_time_to_clock_time
+
+    int_start_time = @start_time.to_i
+    clock_time = int_start_time + @hours_in_school_day
+    if clock_time > 24
```
Comment (robertguturing) [inline lib/school.rb:42]: I really like the consideration of an edge case here

File: test/school_test.rb [hunk 5]
```diff
@@ -45,11 +45,47 @@ def test_it_can_add_student_names
...
+
+    school1 = School.new('8:00', 6)
+    school2 = School.new('8:00', 3)
+    
+
+
+    end_time1 = '2:00'
+    end_time2 = '11:00'
+
+    assert_equal end_time1, school1.convert_end_time_to_clock_time
```
Comment (robertguturing) [inline test/school_test.rb:88]: Nitpick: A good rule of thumb in specs is to have one assertion per test

**Review by robertguturing — COMMENTED** [review]
Thanks for you hard work on this!

Score : 4
